pages/docs/manual/latest/import-from-export-to-js.mdx

```diff
@@ -70,6 +70,25 @@ console.log(Student.default);
 
 </CodeTab>
 
+### Import a JSON file content
+
+Use the `bs.module` [external](external.md):
```

Comment:
Very minor: Since the bs.module and link is mentioned earlier in the page, what do you think about replacing this with a simple sentence such as "ReScript also provides built in support for reading JSON files" (similar to the couple of previous examples).